CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## X.Y.Z] - TODO
```

Comment:
Like this because that's going to change anyway because of https://github.com/mozilla-releng/bouncerscript/pull/53